.github/workflows/release.yml

```
@@ -0,0 +1,44 @@
+name: Release (Publish to Maven Central)
+
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)